www/src/examples/Card/BgColor.js

```diff
@@ -1,97 +1,22 @@
-<>
-  <Card bg="primary" text="white" style={{ width: '18rem' }}>
+[
+  'primary',
+  'secondary',
+  'success',
+  'danger',
+  'warning',
+  'info',
+  'light',
+  'dark'
+].map((variant, idx) => (
+  <><Card bg={variant} text={variant=='light' ? 'dark' : 'white'}  style={{ width: '18rem' }}>
     <Card.Header>Header</Card.Header>
     <Card.Body>
-      <Card.Title>Primary Card Title</Card.Title>
+      <Card.Title>{variant} Card Title </Card.Title>
```

Comment:
Take a look at `Primary` is in titled-case but `variant` will put the value here in lower-case.